src/geometry/manifold/derivation_bundle.lean

```diff
@@ -0,0 +1,163 @@
+/-
+Copyright © 2020 Nicolò Cavalleri. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Nicolò Cavalleri
+-/
+
+import geometry.manifold.algebra.smooth_functions
+import ring_theory.derivation
+
+/-!
+
+# Derivation bundle
+
+In this file we define the derivations at a point of a manifold on the algebra of smooth fuctions.
```

Comment:
```suggestion
In this file we define the derivations at a point of a manifold on the algebra of smooth functions.
```

Reply:
This does not trigger the 100 characters line to me!